examples/src/main/java/io/numaproj/numaflow/examples/sink/simple/SimpleSink.java

```diff
@@ -18,13 +16,13 @@
 @Slf4j
 public class SimpleSink extends SinkHandler {
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws Exception {
```

Comment:
👍

Reply:
I think this was introduced by my recent server writer changes. Could you do me a favour and make the same updates across all the Function examples? Appreciate it.

Reply:
done